pyuptimerobot/decorator.py

```diff
@@ -25,22 +26,28 @@ def decorator(func):
         async def wrapper(*args, **kwargs):
             """Wrapper"""
             client: UptimeRobot = args[0]
-            request_data = f"api_key={client._api_key}&format=json"
             url = f"{API_BASE_URL}{api_path}"
-            if kwargs:
-                for key, value in kwargs.items():
-                    request_data += f"&{key}={value}"
-            LOGGER.debug("Requesting %s", url)
+            if "monitor_id" in kwargs:
+                url = url.format(monitor_id=kwargs.pop("monitor_id"))
+            headers = {
+                "Authorization": f"Bearer {client._api_key}",
+                **API_HEADERS,
```

Comment:
```suggestion
                "Content-Type": "application/json",
```
As you are deconstructing on each call, this is better.
Ideally also just set it directory on the request call.